lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_generate_po_file_from_metadata.rb

```diff
@@ -0,0 +1,145 @@
+require_relative '../../helper/po_extended'
+require 'fastlane/action'
+require 'gettext/po'
+
+module Fastlane
+  module Actions
+    class IosGeneratePoFileFromMetadataAction < Action
+      # TODO: release_notes_previous.txt must be generated by the code_freeze lanes
+
+      # SPECIAL_KEYS are keys that need to be treated specially
+      SPECIAL_KEYS = %w[release_notes release_notes_previous].freeze
+      REQUIRED_KEYS_TO_COMMENT_HASH = {
+        release_notes: 'Multi-paragraph text used to display in the Play Store. Limit to 4000 characters including spaces and commas!',
+        name: 'The application name in the Apple App Store. Please keep the brand names (\'Jetpack\' and WordPress\') verbatim. Limit to 30 characters including spaces and punctuation!',
+        subtitle: 'Subtitle to be displayed below the application name in the Apple App Store. Limit to 30 characters including spaces and commas!',
+        description: 'Multi-paragraph text used to display in the Apple App Store.',
+        keywords: 'Keywords used in the App Store search engine to find the app.
+.Delimit with a comma between each keyword. Limit to 100 characters including spaces and commas.',
+        release_notes_previous: nil
+      }.freeze
+
+      REQUIRED_KEYS = REQUIRED_KEYS_TO_COMMENT_HASH.keys.map(&:to_s).freeze
+      # rubocop: disable Naming/VariableNumber
+      OPTIONAL_KEYS_TO_LABELS_HASH = {
```

Comment:
@jkmassel @AliSoftware @mokagio 
if I wanted to take this  `OPTIONAL_KEYS_TO_LABELS_HASH` out of this class so that it can be also accessed by `AndroidGeneratePoFileFromMetadataAction`, what would be a good way to do it in Ruby?

Reply:
Makes sense.
However given the biggest problem being that `OPTIONAL_KEYS_TO_LABELS_HASH` should not be hardcoded this question is somewhat outdated.